app/Http/Controllers/Api/V1/UserController.php

```diff
@@ -218,9 +218,33 @@ public function store(Request $request)
     /**
      * Create New Organization User
      *
-     * Create a new user in storage.
+     * Create a new organization user in storage.
+     *
+     * @urlParam suborganization integer required The Id of an suborganization Example: 1
+     * @bodyParam first_name string required First name of a user Example: John
+     * @bodyParam last_name string required Last name of a user Example: Doe
+     * @bodyParam email string required Email of a user Example: doe@gmail.com
+     * @bodyParam password string required Password of a user
+     * @bodyParam role_id integer required Role Id of a user Example: 1
+     * @bodyParam organization_name string Organization name of a user Example: Curriki
+     * @bodyParam organization_type string Type of an organization Example: K-12
+     * @bodyParam website string Website url of a user Example: www.currikistudio.org
+     * @bodyParam job_title string Job title of a user Example: Developer
+     * @bodyParam address string Address of a user Example: 20660 Stevens Creek Blvd #332, Cupertino, CA 95014
+     * @bodyParam send_email boolean true or false for email sending Example: true
```

Comment:
"T" of "true or false..." should be capital.

Reply:
Done!